bin/librato-storm-kafka

```diff
@@ -236,13 +236,18 @@ def do_thr()
     break if work.nil?
 
     begin
-      work = monitor_partition(work)
+      result = monitor_partition(work)
     rescue Errno::ETIMEDOUT, Errno::EHOSTUNREACH => e
```

Comment:
Possibly, guess I'd like to see specifically why it's failing? If I can get Honeybadger hooked up to this then we can let exceptions fall through and we'll get the HB report email.